xbmc/pvr/timers/PVRTimers.cpp

```diff
@@ -546,7 +595,7 @@ bool CPVRTimers::DeleteTimer(const CFileItem &item, bool bForce /* = false */)
   if (!tag)
     return false;
 
-  return tag->DeleteFromClient(bForce);
+  return tag->DeleteFromClient(bForce,bDeleteSchedule);
```

Comment:
@ksooo fully cool your changes, wow. Thanks a lot. On fast review it looks very good. Maybe only here a space:-) .